test-suite/output/Arguments.out

```diff
@@ -43,14 +43,14 @@ forall D1 C1 : Type,
 (D1 -> C1) -> forall D2 C2 : Type, (D2 -> C2) -> D1 * D2 -> C1 * C2
 
 pf is not universe polymorphic
-Arguments pf {D1%foo_scope} {C1%type_scope} _ [D2] [C2] : simpl never
+Arguments pf {D1}%foo_scope {C1}%type_scope _ [D2 C2] : simpl never
```

Comment:
When there is a single argument we have a choice between putting the scope inside vs outside the brackets, I decided to have it outside since that's what happens when they get combined.